exercises/5a.sql

```diff
@@ -1 +1,5 @@
 -- How many tracks are longer than 5 minutes?
+
+SELECT COUNT(*)
+FROM tracks
+WHERE duration_ms / 60000 > 5;
```

Comment:
Nice, you converted it into minutes first and then checked, I did it a bit differently and left it in ms. They both work!